### PR TITLE
either display granger or adv granger in evo menu

### PR DESCRIPTION
**Pull request by Gireen**
Only one of them is needed at the same time depending on if the adv granger is unlocked.

### CODE REVIEW

**Comment by illwieckz** [comment]
looks good

File: src/cgame/cg_rocket_datasource.cpp [hunk 1]
```diff
@@ -1647,6 +1647,12 @@ void CG_Rocket_BuildAlienEvolveList( const char *table )
 		{
 			if ( BG_Class( i )->team == TEAM_ALIENS )
 			{
+				if ( ( i == PCL_ALIEN_BUILDER0 && ( BG_ClassUnlocked( PCL_ALIEN_BUILDER0_UPG ) && !BG_ClassDisabled( PCL_ALIEN_BUILDER0_UPG ) ) )||
```
Comment (DolceTriade) [inline src/cgame/cg_rocket_datasource.cpp]: I believe that we should not include this change in the "backend". This is a strictly UI change that should be made in the UI...The circleMenu can use a filter function or something that can filter things out like this...

Comment (Gireen) [reply src/cgame/cg_rocket_datasource.cpp]: Is CG_Rocket_DFCMAlienEvolve the right place for it?

Comment (DolceTriade) [reply src/cgame/cg_rocket_datasource.cpp]: Yeah, it's fine to put it there.

**Comment by illwieckz** [comment]
@DolceTriade is it ok for you now?

**Pull request merged**